trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoPlugin.java

```diff
@@ -10,7 +10,7 @@
 
 /** Trino plugin endpoint, using java spi mechanism */
 public class GravitinoPlugin implements Plugin {
-
+  @Override
```

Comment:
Can you please add a blank line above?

Reply:
Updated. Please help to review again when you have time, thanks.